src/Sanitization/filters.js

```diff
@@ -109,17 +109,19 @@ SanitizationFilters.normalizeEmail = function (value, args) {
     removeExtension: true
   }
 
-  args.forEach(function (option) {
-    if (option === '!lc') {
-      options.lowercase = false
-    }
-    if (option === '!rd') {
-      options.removeDots = false
-    }
-    if (option === '!re') {
-      options.removeExtension = false
-    }
-  })
+  if(typeof args === 'object') {
```

Comment:
Can u change the check to be more specific to array. `if(args instanceof Array)`